Week-1/exercise2.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css" integrity="sha512-rSWTr6dChYCbhpHaT1hg2tf4re2jUxBWTuZbujxKg96+T87KQJriMzBzW5aqcb8jmzBhhNSx4XYGA6/Y+ok1vQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+        <style>
+            body{
+                margin: 0;
+                padding: 0;
+            }
+            .container{
+                width: 100%;
+                height: 100%;
+            }
+            .sidebar {
+                height: 100%;
+                width: 50px;
+                position: fixed;
+                z-index: 1;
+                top: 0;
+                left: 0;
+                background-color:#f0f0f0;
+                overflow-x: hidden;
+                padding-top: 16px;
+            }
+            .sidebar a {
+                padding: 6px 8px 6px 16px;
+                text-decoration: none;
+                font-size: 20px;
+                color: black;
+                display: block;
+            }
+            .sidebar a i{
+                line-height: 4.5;
+            }
+            .right-content{
+                padding-left: 50px;
+            }
+            .info-board{
+                display: grid;
+                grid-template-columns: 30% 70%;
+                background-color: #454545;
+            }
+            .info-board-left{
+                padding: 50px;
+            }
+            .info-board-right{
+                padding-top: 30px;
+                color: white;
+                position: relative;
+            }
+            .info-board-right p{
+                line-height: 0%;
+                
+            }
+            .info-board-right .board-actions{
+                position: absolute;
+                bottom: 0;
+                width: 100%;
+                padding-bottom: 6%;
+            }
+            .board-actions a{
+                text-decoration: none;
+                color: white;
+                padding: 2%;
+            }
+            .board-actions a i{
+                padding-right: 1%;
+            }
+            .main-content-container{
+                display: grid;
+                grid-template-columns: auto auto auto auto;
+                padding: 10px;
+            }
+            .grid-item{
+                height: 50px;
+                padding-top: 25px;
+                padding-left: 10px
+            }
+            .even {
+                background-color: rgba(255, 255, 255, 0.8);
+               
+            }
+            .odd {
+                background-color: rgb(240 240 240);
```

Comment:
why do we have different ways fo colors. The Coding should be uniform. We should use # color codes generally